src/struct_info.json

```diff
@@ -1629,5 +1629,283 @@
             "EMSCRIPTEN_FETCH_SYNCHRONOUS",
             "EMSCRIPTEN_FETCH_WAITABLE"
         ]
+    },
+    // ===========================================
```

Comment:
Did you generate this one using Dawn's generators? Can we land the generator + template in Dawn so updating the list just requires running that build target and copy-pasting?

Reply:
Yes, I generated it using your old patch, though with minor manual edits for JSON validity. I'll work on landing it.

Reply:
WIP https://dawn-review.googlesource.com/c/dawn/+/15240